src/types.ts

```diff
@@ -26,7 +26,28 @@ export interface AragonConfig {
    * ```
    */
   appName?: string | { [network: string]: string }
+
+  /**
+   * IPFS gateway, to fetch static files such as the artifact
+   * of your app's latest version
+   * Examples:
+   * - http://localhost:8080
+   * - https://ipfs.io
+   *
+   * Defaults to: 'https://ipfs.eth.aragon.network/ipfs/'
```

Comment:
Just wondering if we can document these configuration options a bit better (especially so if we expose them to the user).

From what I understand, we only use the `ipfsGateway` to _pull_ data, and only data from other networks (i.e. for the install feature). We use the `ipfsApi` to _push_ data, when publishing.

To eyes without context, it is very easy to start asking questions why we're using a local API but a remote gateway. As I understand it, we never need to pull data locally, since the local app's frontend is always being hosted via http, and all other installed apps' frontends should be fetched from the remote gateway?

Reply:
Addressed in https://github.com/aragon/buidler-aragon/pull/121